lab3/src/task4/Lindt.java

```diff
@@ -0,0 +1,56 @@
+package task4;
+
+public class Lindt extends CandyBox {
+    private float length, width, height;
```

Comment:
Where is the type? :) Also, even though they have the same type, you must declare them separately to be easier to see what are the class fields